### PR TITLE
Fix events

### PR DESCRIPTION
**Pull request by philipstanislaus**

### CODE REVIEW

File: types/event_record.go [hunk 1]
```diff
@@ -114,6 +115,27 @@ type EventImOnlineHeartbeatReceived struct {
 	Topics      []Hash
 }
 
+type Exposure struct {
```
Comment (vedhavyas) [inline types/event_record.go]: Can you add a comment to the exported structs pls

File: types/event_record.go [hunk 2]
```diff
@@ -114,6 +115,27 @@ type EventImOnlineHeartbeatReceived struct {
...
+	Total  UCompact
+	Own    UCompact
+	Others []IndividualExposure
+}
+
+type IndividualExposure struct {
```
Comment (vedhavyas) [inline types/event_record.go]: same as above

File: Gopkg.lock [hunk 3]
```diff
@@ -1,6 +1,17 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  branch = "development"
+  digest = "1:ad77b20c92f2418802e50880d27084689d742fe263735b239aff52a713720627"
+  name = "github.com/ChainSafe/gossamer"
```
Comment (vedhavyas) [inline Gopkg.lock]: is this not part of Gopkg.toml already ?

File: Gopkg.lock [hunk 4]
```diff
@@ -1,6 +1,17 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  branch = "development"
```
Comment (vedhavyas) [inline Gopkg.lock]: a specific hash would be preferrable

File: types/event_record.go (hunk 2 above)
Comment (philipstanislaus) [reply types/event_record.go]: Done

File: types/event_record.go (hunk 1 above)
Comment (philipstanislaus) [reply types/event_record.go]: Done

File: Gopkg.lock (hunk 3 above)
Comment (philipstanislaus) [reply Gopkg.lock]: Good catch, that sneaked in because of a local experiment

**Pull request merged**